bin/dev-pgadmin

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env bash
 
+rm -rf /tmp/servers.json
```

Comment:
(blocking) A few issues
- This is in the wrong spot, it should be below `docker::set_project_name_dev` line.
- We should not be using `rm -rf` in scripts that making changes to a user's filesystem outside the project directory.
- We should revamp how this whole thing works. Whatever the issue is that is sometimes causing this to become a directory concerns me. For now I'm fine with a targeted check to remove the specific directory like this `[[ -d "/tmp/servers.json" ]] && rmdir /tmp/servers.json`. It should be sufficient since it's an empty directory when it's happened to me.

Reply:
Done! Thank you for your input.